pom.xml

```diff
@@ -43,7 +43,7 @@
         <groovy-all.version>3.0.6</groovy-all.version>
 
         <!-- org.apache.maven.plugins -->
-        <maven-enforcer-plugin-version>3.0.0-M2</maven-enforcer-plugin-version>
+        <maven-enforcer-plugin-version>3.0.0-M3</maven-enforcer-plugin-version>
```

Comment:
Change is here.

Reply:
For consistent with axm maven-enforcer-plugin-version. Nevermind, I've reverted back it.